docs/topics/wasm/wasm-get-started.md

```diff
@@ -12,7 +12,7 @@ app with Kotlin/Wasm in IntelliJ IDEA, and generate artifacts to publish as a si
 Create a project using the Kotlin Multiplatform wizard:
 
 1. Open the [Kotlin Multiplatform wizard](https://kmp.jetbrains.com/#newProject).
-2. On the **New Project** tab, change the project name to "WasmDemo" and the project ID to "wasm.project.demo".
+2. On the **New Project** tab, you can change the project name to "WasmDemo" and the project ID to "wasm.project.demo".
```

Comment:
Similar to what I said in the debugging tutorial, I think "you can" should be removed so it remains an instruction. Otherwise it could be interpreted as  a suggestion which would break later steps.

Reply:
If they don't do it, it doesn't break later steps. This is actually optional. They can leave it as it is or add the name to their preference. This change is due to some feedback from the SME saying that it needed to be clarified that this is optional. I modified the step and added the clarification in a separate note. What do you think?